app/controllers/anonymous_feedback/export_requests_controller.rb

```diff
@@ -33,11 +34,20 @@ def export_request_params
   end
 
   def anonymous_feedback_params
-    @anonymous_feedback_params ||= params.permit(:from, :to, :path, :organisation).to_h
+    clean_paths
+    @anonymous_feedback_params ||= params.permit(:from, :to, :organisation, :path, paths: []).to_h
```

Comment:
I think we always send `paths` as a comma separated string now so we should be permitting paths as a scalar (`:paths`) not as an array (`paths: []`).  I'm surprised a test hasn't failed to expose this as I think without this the controller would see a param like `paths=/foo,/bar` as `nil`.

Reply:
`clean_paths` will split `paths` into an array, before going through the param filtering. So `paths` is guaranteed to be an array every time.

Reply:
You're right.  My bad.